core/system/sys_stat.go

```diff
@@ -0,0 +1,73 @@
+package system
+
+import (
+	"sync/atomic"
+	"time"
+
+	"github.com/shirou/gopsutil/cpu"
+	"github.com/shirou/gopsutil/load"
+)
+
+const (
+	notRetrievedValue float64 = -1
+)
+
+var (
+	currentLoad     atomic.Value
+	currentCpuUsage atomic.Value
+
+	ssStopChan = make(chan struct{})
+)
+
+func init() {
+	currentLoad.Store(notRetrievedValue)
+	currentCpuUsage.Store(notRetrievedValue)
+
+	ticker := time.NewTicker(1 * time.Second)
+	go func() {
```

Comment:
In order to avoid sentinel internal panic interrupt main progress. It may be better to use auto recover goroutine.
Refer to PR: https://github.com/sentinel-group/sentinel-golang/pull/31/files

Reply:
Good idea. I've polished the code, please check.